code/modules/atmospherics/machinery/portable/portable_atmospherics.dm

```diff
@@ -48,7 +48,7 @@
 		return 0
 
 	//Make sure are close enough for a valid connection
-	if(new_port.loc != loc)
+	if(new_port.loc != loc && new_port.loc != loc.loc)

```

Comment:
Please explain this and consider using get_turf()

Reply:
I just dont know how to force 'proc/connect' work from the inside of mech.
Do you mean someshing like:
`if(new_port.loc != loc && new_port.loc != loc.get_turf())`
?